sdk/core/core-amqp/CHANGELOG.md

```diff
@@ -4,6 +4,8 @@
 
 ### Features Added
 
+- Adds support for connecting to the development emulator. The connection string for the development emulator should have the `";UseDevelopmentEmulator=true"` slug.
```

Comment:
Can it not start with `UseDevelopmentEmulator=true`?

Reply:
The slug could appear anywhere in the connection string and it is not required to be at the beginning.